step.yml

```diff
@@ -344,14 +344,14 @@ outputs:
     opts:
       title: The created .dSYM dir's path
       description: |-
-        If `export_all_dsyms` is set to `"yes"`, step will collect every dsym (.app dsym and framwork dsyms) in a directory and export the directory path.
-        Otherwise this environment points to the .app.DSYM directory path.
+        This environment points to the path of the directory which contains the dSYMs files.
```

Comment:
This Environment Variable points to the path of the directory which contains the dSYMs files.